data/util.py

```diff
@@ -73,24 +86,43 @@ def get_dds_config(user):
     :param user: A Django model user object
     :return: ddsc.config.Config: settings to use with ddsclient
     """
-    config = Config()
-
-    # Get our agent key
-    app_cred = DDSEndpoint.objects.first()
-    config.update_properties({'agent_key': app_cred.agent_key})
-    config.update_properties({'url': app_cred.api_root})
-
-    # Setup user key if exists, otherwise setup dds temporary auth token
     try:
         user_cred = DDSUserCredential.objects.get(user=user)
+        return get_dds_config_for_credentials(user_cred)
         config.update_properties({'user_key': user_cred.token})
```

Comment:
Was this line supposed to be deleted? Also, the `return` and `config` variable lifecycle isn't consistent.

Reply:
The update_properties method was moved out to get_dds_config_for_credentials. I cleaned this method up.